config/config.go

```diff
@@ -72,6 +76,16 @@ func (cc *Config) SecurityConfig() *SecurityConfig {
 	return cc.securityConfig
 }
 
+// ReliableTopicConfig returns the reliable topic config for this client.
+func (cc *Config) ReliableTopicConfig(name string) *ReliableTopicConfig {
```

Comment:
Naming is not consistent with `GetFlakeIDGeneratorConfig` 
Which one is correct ?

Reply:
I think implementation also should be similar. Using the `default` when not found is missing here.

Reply:
We can use `GetReliableTopicConfig` since it is not a getter. This one creates a new one with the given name when it is not found.  It is the same as Java Client.

Reply:
Oh, java client also does not use `default`. But on the member config we use `default` when given name is not found. Can you open an issue for java client also to fix this ?
And can you also update this so that it behaves similar to current member ?
https://github.com/hazelcast/hazelcast/blob/master/hazelcast/src/main/java/com/hazelcast/config/Config.java#L1881